apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -395,7 +400,10 @@ internal SubscriberClientImpl(SubscriptionName subscriptionName, IEnumerable<Sub
             settings.Validate();
             // These values are validated in Settings.Validate() above, so no need to re-validate here.
             _modifyDeadlineSeconds = (int)((settings.AckDeadline ?? DefaultAckDeadline).TotalSeconds);
-            _autoExtendInterval = TimeSpan.FromSeconds(_modifyDeadlineSeconds) - (settings.AckExtensionWindow ?? DefaultAckExtensionWindow);
+            var autoExtendInterval = TimeSpan.FromSeconds(_modifyDeadlineSeconds) - (settings.AckExtensionWindow ?? DefaultAckExtensionWindow);
+            // Ensure the duration between lease extentions is at least MinimumLeaseExtensionDelay (5 seconds).
```

Comment:
(Suggestions aren't working for me today for some reason.)

extentions => extensions

Reply:
Oops! Done